apps/notifications/src/panel/templates/undo-list-item.jsx

```diff
@@ -174,6 +174,14 @@ export class UndoListItem extends Component {
 		this.instance = ref;
 	};
 
+	hideNotification = ( event ) => {
+		if ( event ) {
+			event.preventDefault();
+		}
+		this.instance && this.setState( { isVisible: false } );
+		clearTimeout( this.state.undoTimer );
```

Comment:
`actImmediately` function have similar code so let's remove that and call `this.hideNotification` instead.

If needed then as a precaution we can change `if ( event )` to `if ( event && event.preventDefault )`.

Reply:
Thanks for pointing that out! I decided to keep `actImmediately` and prevent the panel close by adding `event.stopPropagation();`. I don't think we need the `event.preventDefault` check, since it's only being called on this click. Update in https://github.com/Automattic/wp-calypso/pull/97850/commits/6904b32785470a802513cd7ea76213c7daec0f86